<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileDataPlaylistWidgetUI</class>
 <widget class="QWidget" name="ProfileDataPlaylistWidgetUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>126</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_format">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Format:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KComboBox" name="kcombobox_format"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_pattern">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Name/Pattern:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KLineEdit" name="klineedit_pattern"/>
   </item>
   <item row="1" column="2">
    <widget class="KPushButton" name="kpushbutton_pattern"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBox_abs_file_path">
     <property name="text">
      <string>Use absolute file paths in playlist</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBox_utf8">
     <property name="text">
      <string>UTF-8 coding in playlist</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>4</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
